ui/components/subscription/TeamMetadataModal.tsx

```diff
@@ -5,20 +5,71 @@ import { Widget } from '@typeform/embed-react'
 import { DEFAULT_CHAIN, TEAM_TABLE_ADDRESSES } from 'const/config'
 import { useRouter } from 'next/router'
 import { useCallback, useEffect, useState } from 'react'
+import toast from 'react-hot-toast'
 import { pinBlobOrFile } from '@/lib/ipfs/pinBlobOrFile'
 import { unpin } from '@/lib/ipfs/unpin'
 import { createSession, destroySession } from '@/lib/iron-session/iron-session'
 import cleanData from '@/lib/tableland/cleanData'
 import deleteResponse from '@/lib/typeform/deleteResponse'
-import formatTeamFormData from '@/lib/typeform/teamFormData'
 import waitForResponse from '@/lib/typeform/waitForResponse'
 import { renameFile } from '@/lib/utils/files'
 import { getAttribute } from '@/lib/utils/nft'
+import FormInput from '../forms/FormInput'
 import Modal from '../layout/Modal'
 import { ImageGenerator } from '../onboarding/TeamImageGenerator'
 import { PrivyWeb3Button } from '../privy/PrivyWeb3Button'
 import DeleteProfileData from './DeleteProfileData'
 
+function TeamMetadataForm({ teamData, setTeamData }: any) {
+  return (
+    <div className="w-full flex flex-col gap-2">
+      <FormInput
+        label="Name"
+        value={teamData.name}
+        onChange={({ target }: any) =>
+          setTeamData((prev: any) => ({ ...prev, name: target.value }))
+        }
+        placeholder="Enter your name"
+      />
+      <FormInput
+        label="Bio"
+        value={teamData.description}
+        onChange={({ target }: any) =>
+          setTeamData((prev: any) => ({ ...prev, description: target.value }))
+        }
+        placeholder="Enter your bio"
+      />
+      <FormInput
+        label="Twitter"
+        value={teamData.twitter}
+        onChange={({ target }: any) =>
+          setTeamData((prev: any) => ({ ...prev, twitter: target.value }))
+        }
+        placeholder="Enter your twitter link"
+      />
+      <FormInput
+        label="Communications"
+        value={teamData.communications}
+        onChange={({ target }: any) =>
+          setTeamData((prev: any) => ({
+            ...prev,
+            communications: target.value,
+          }))
+        }
+        placeholder="Enter your communications link"
```

Comment:
Communications Link is a little vague... Maybe Ryan has a better word for this.
Something like "Other socials Link"?